packages/sommelier.rb

```diff
@@ -47,246 +39,281 @@ class Sommelier < Package
     @peer_cmd_prefix = '/lib64/ld-linux-x86-64.so.2'
   end
 
+  def self.preflight
+    unless File.socket?('/var/run/chrome/wayland-0')
+      abort "This package is not compatible with your device :/".lightred
+    end
+  end
+
+  def self.prebuild
```

Comment:
Why this instead of self.patch?

Reply:
According to the `Manual.md`:
```
self.prebuild - Only sed commands should be run during this stage - if provided
```
(but `self.patch` 's meaning is more accurate :) )

Reply:
Changed to `self.patch`